core/build.gradle

```diff
@@ -100,6 +100,9 @@ dependencies {
     // Caffeine
     implementation libs.caffeine
 
+    // https://mvnrepository.com/artifact/net.openhft/zero-allocation-hashing
+    implementation 'net.openhft:zero-allocation-hashing:0.16'
```

Comment:
What do you think of forking https://github.com/OpenHFT/Zero-Allocation-Hashing/blob/ea/src/main/java/net/openhft/hashing/XxHash.java to our code base or shading and timing the library? 
- We only use `XxHash` but Zero-Allocation-Hashing has additional hash functions.
- We don't expose a utility library as a transitive dependency in the POM file.